nemo/collections/nlp/data/language_modeling/megatron/gpt_prompt_learning_dataset.py

```diff
@@ -122,8 +123,9 @@ def load_data(self, dataset):
         """
         skipped = 0
 
+        ex_id = -1
```

Comment:
Can we remove all the ex_id stuff?

Reply:
it is nice to have the actual example id for debugging things like this. ofc, its easy to remove if you feel strongly.